openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py

```diff
@@ -95,6 +96,12 @@ def build_source(ingestion_pipeline: IngestionPipeline) -> WorkflowSource:
         service: MlModelService = metadata.get_by_name(
             entity=MlModelService, fqn=ingestion_pipeline.service.name
         )
+    elif service_type == "TestSuite":
```

Comment:
nit - Maybe we can move this check to the top? I am imagining the flow like if we are a TestSuite there's nothing else to check. Otherwise, we need to pick the right service, validate its not null... Kind of just to group the reading a bit